index.js

```diff
@@ -54,7 +54,11 @@ module.exports = fp(async function (app, opts) {
       query: new GraphQLObjectType({
         name: 'Query',
         fields: { }
-      })
+      }),
+      mutation: opts.defineMutation ? new GraphQLObjectType({
+        name: 'Mutation',
+        fields: { }
+      }) : undefined
```

Comment:
Why are you doing this only behind an option? Probably we should just do this all the time?

Reply:
I chose the option way because otherwise it would force the user to define at least one mutation, which may not be required for every use case.